examples/azure-monitoring/eventhub_integration/variables.tf

```diff
@@ -0,0 +1,7 @@
+variable "prefix" {
```

Comment:
As this is not used in your example, can we remove this file?

Reply:
Thanks for the comments, I removed the file